packages/account/codegen.json

```diff
@@ -22,7 +22,8 @@
           "AssetId": "string",
           "ContractId": "string",
           "Salt": "string",
-          "Nonce": "string"
+          "Nonce": "string",
+          "Tai64Timestamp": "string"
```

Comment:
@petertonysmith94 I believe we can also remove this since it has no use for the `graphql-codegen`

Reply:
Hey @Torres-ssf I'd disagree with this one. Without the `Tai64Timestamp` we would get a type of `any` without our generated operations. We know the `Tai64Timestamp` is a `string`, so why not type hint it?

Reply:
@petertonysmith94 You're right. 

I did forget that `Tai64Timestamp` is already a Scalar defined on the GraphQL server. 

Thanks for clarifying that.

Reply:
@Torres-ssf no worries at all, thank you for your review!